static_src/components/loading.jsx

```diff
@@ -0,0 +1,25 @@
+
+import React from 'react';
+
+const propTypes = {
```

Comment:
Is there a reason you defined these into separate variables rather then directly like other components:

```
Loading.proptTypes = {
  text
  ...
```

Is this a better pattern we should adopt?